main/musl/patches/getdents.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/dirent/posix_getdents.c b/src/dirent/posix_getdents.c
```

Comment:
i sent this in https://www.openwall.com/lists/musl/2024/05/12/2 for reference

Reply:
ty